drivers/sdhc/rcar_mmc.c

```diff
@@ -496,6 +504,9 @@ static int rcar_mmc_dma_rx_tx_data(const struct device *dev,
 	uint32_t reg;
 	int ret = 0;
 	uint32_t dma_info1_poll_flag;
+#ifdef CONFIG_RCAR_MMC_DMA_IRQ_DRIVEN_SUPPORT
+	struct mmc_rcar_data *dev_data = dev->data;
```

Comment:
Note: do we actually need this construction just to save 1 reference here?
`ret = k_sem_take(&dev_data->irq_xref_fin, ...`